backend-API-routes/backend-api-routes_products/pages/api/products/[id].js

```diff
@@ -0,0 +1,7 @@
+import { getProductById } from "../../../services/productServices.js";
+import products from "../../../services/productServices.js";
+
+export default function handler(request, response) {
+    const id = request.query.id;
```

Comment:
Just as an alternative, here you can also use destructuring:     const {id} = request.query; But the outcome is the same.